static/app/views/organizationDetails/body.tsx

```diff
@@ -100,6 +101,7 @@ function DeletionPending({organization}: Props) {
 
 function OrganizationDetailsBody({children, organization}: Props) {
   const status = organization?.status?.id;
+  const routeContext = useRouteContext();
```

Comment:
this is kind of unrelated but I noticed that we were rendering this footer below the new footer and when scrolling it was bugging me.